doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -2,64 +2,52 @@
  Mount CephFS using Kernel Driver
 =================================
 
-Prerequisite
-------------
-Before mounting CephFS, copy the Ceph configuration file and keyring for the
-CephX user that has CAPS to mount MDS to the client host (where CephFS will be
-mounted and used) from the host where Ceph Monitor resides. Please note that
-it's possible to mount CephFS without conf and keyring, but in that case, you
-would have to pass the MON's socket and CephX user's secret key manually to
-every mount command you run.
+To make a choice between kernel and FUSE mounted CephFS and to go through
+the prerequisites required by both read `Mount CephFS: Prerequisites`_ page.
 
-#. Generate a minimal conf file for the client host and place it at a
-   standard location::
+Prerequisites
+=============
 
```

Comment:
In the rendered docs, it's weird to have a section title followed immediately by a sub-section title. We can put the link and warning to "Mount CephFS: Prereq..." here.

Reply:
Agreed, that's much better.

Reply:
Move "Mount CephFS: Prerequisites" part.